catalogue_pipeline/ingestor/src/test/scala/uk/ac/wellcome/platform/ingestor/services/WorkIndexerTest.scala

```diff
@@ -83,20 +89,188 @@ class WorkIndexerTest
       }
     }
   }
+  describe("updating merged / redirected works") {
+    it(
+      "doesn't override a merged Work with same version but merged flag = false") {
+      val mergedWork = createIdentifiedWorkWith(version = 3, merged = true)
+      val unmergedWork = mergedWork.copy(merged = false)
+
+      withLocalElasticsearchIndex { indexName =>
```

Comment:
There's a lot of repeated code here. Could it be pulled into a common function?

Something like:

```scala
assertCorrectWorkIsIndexed(
    successfulWork: Work,
    rejectedWork: Work
)
```

Reply:
It would also be easier to read the list of current test cases, and see if we're missing any.

For example: “a merged work is replaced by an unmerged work with a higher version”.